src/wallet/load.cpp

```diff
@@ -71,11 +71,16 @@ bool VerifyWallets(interfaces::Chain& chain)
 
         DatabaseOptions options;
         DatabaseStatus status;
+        options.require_existing = true;
         options.verify = true;
         bilingual_str error_string;
         if (!MakeWalletDatabase(wallet_file, options, status, error_string)) {
-            chain.initError(error_string);
-            return false;
+            if (status == DatabaseStatus::FAILED_NOT_FOUND) {
+                chain.initWarning(Untranslated(strprintf("Skipping -wallet path that doesn't exist. %s\n", error_string.original)));
```

Comment:
bc2b0d99a5f54b941f8c663385f42cb9da20b94b

This is a user faced message. Why keep it untranslated?

Reply:
> This is a user faced message. Why keep it untranslated?

It's a technical message not going to be shown to a typical GUI user just loading and unloading wallets, instead of manually moving files inside wallet directories or creating a bitcoin.conf and writing a list of wallets. I think the message might also be short lived (https://github.com/bitcoin-core/gui/issues/95#issuecomment-694236940) and not worth translating right away. But I don't have a strong opinion and would be happy to change. I believe similar warnings like ones from UpdateWalletSetting are also untranslated.

Reply:
Wallets are just files, and such a warning could be caused by user's operations with wallets.

>  I believe similar warnings like ones from UpdateWalletSetting are also untranslated.

Correct. I think the reason of that is an absence of the pr :)
Sure, no need to change here.

Reply:
> Wallets are just files, and such a warning could be caused by user's operations with wallets.

Sure. If this warning turns out to occur by common file operations, my first preference would be to implement one of the other alternatives in https://github.com/bitcoin-core/gui/issues/95#issuecomment-694236940 so the warning stops occurring